### PR TITLE
Count pages with uploaded files

### PR DESCRIPTION
**Pull request by MaxSem**
Bug: https://phabricator.wikimedia.org/T215356

### CODE REVIEW

File: src/AppBundle/Service/EventProcessor.php [hunk 1]
```diff
@@ -380,9 +384,13 @@ private function setFilesUploaded(EventWiki $wiki, EventWikiRepository $ewRepo):
         $this->createOrUpdateEventWikiStat($wiki, 'files-uploaded', $ret);
         $this->filesUploaded += $ret;
 
-        $ret = $this->eventRepo->getFileUsage($dbName, $start, $end, $this->getParticipantNames());
+        $ret = $this->eventRepo->getUsedFiles($dbName, $start, $end, $this->getParticipantNames());
         $this->createOrUpdateEventWikiStat($wiki, 'file-usage', $ret);
         $this->fileUsage += $ret;
+
+        $ret = 0;//$this->eventRepo->getPagesUsingFiles($dbName, $start, $end, $this->getParticipantNames());
```
Comment (MusikAnimal) [inline src/AppBundle/Service/EventProcessor.php]: Lingering test code?

File: tests/AppBundle/Repository/EventRepositoryTest.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        $kernel = static::bootKernel();
+        /** @var \Doctrine\ORM\EntityManager $entityManager */
+        $entityManager = $kernel->getContainer()->get('doctrine')->getManager();
+        $repo = new EventRepository($entityManager);
+        $repo->setContainer($kernel->getContainer());
+
+        static::assertGreaterThan(0, $repo->getPagesUsingFile('commonswiki_p', 'Ultrasonic_humidifier.jpg'));
+        static::assertGreaterThan(0, $repo->getPagesUsingFile('enwiki_p', '2-cube.png'));
+    }
+}
```
Comment (MusikAnimal) [inline tests/AppBundle/Repository/EventRepositoryTest.php]: We don't usually test repositories (comment for the EventRepository class is even marked with `@codeCoverageIgnore`), but I'm certainly not opposed to it :smiley: I think it'd be good to have some integration testing. The "extended" fixture includes local file uploads that should always have more than zero pages using the files: https://github.com/wikimedia/eventmetrics/blob/master/src/AppBundle/DataFixtures/ORM/extended.yml#L47-L51 So in ProcessEventCommandTest you should be able to add a `pagesWithFilesSpec()`, like the others, making assertions against the relevant EventStat.

File: src/AppBundle/Service/EventProcessor.php (hunk 1 above)
Comment (MaxSem) [reply src/AppBundle/Service/EventProcessor.php]: Fixed.

File: tests/AppBundle/Command/ProcessEventCommandTest.php [hunk 3]
```diff
@@ -315,6 +315,14 @@ private function fileUsageSpec(): void
...
             ]);
         static::assertGreaterThan(1, $eventWikiStat->getValue());
+
+        $eventWikiStat = $this->entityManager
+            ->getRepository('Model:EventWikiStat')
+            ->findOneBy([
+                'wiki' => $this->event->getWikiByDomain('commons.wikimedia'),
+                'metric' => 'pages-using-files',
+            ]);
+        static::assertGreaterThan(0, $eventWikiStat->getValue());
```
Comment (MusikAnimal) [inline tests/AppBundle/Command/ProcessEventCommandTest.php]: Can we add one for enwiki too? Just for sanity :)

**Pull request merged**